lib/java/spring/kafka/streams/src/main/resources/kafka-streams.properties

```diff
@@ -0,0 +1,6 @@
+kafka.brokers=${KAFKA_BROKERS}
+kafka.schema-registry-url=${KAFKA_SCHEMA_REGISTRY_URL}
+kafka.commit-interval-ms=${KAFKA_COMMIT_INTERVAL_MS:1000}
+
+# Only used for testing:
+kafka.cleanup=${KAFKA_CLEANUP:false}
```

Comment:
How does this works for testing? There are other properties like.
```
kafka.commit-interval-ms=0
kafka.cache.max.bytes=0
kafka.delete.retention.ms=0
```
There are still taken into account?

Reply:
Yes they are still taken into account. The only thing that changed is that these values are now mapped in a single place. Test properties are still a hard override.